src/WCSimPhysicsList.cc

```diff
@@ -392,349 +369,341 @@ void WCSimPhysicsList::ConstructGeneral()
+ 	  pmanager->AddDiscreteProcess(theElasticProcess);
+	  G4DeuteronInelasticProcess* theInelasticProcess =
+ 	    new G4DeuteronInelasticProcess();
+
+	  // Use INCL++ for low energy and FTFP for high energy - SS
+	  G4INCLXXInterface* theINCLXXModel = new G4INCLXXInterface();
+	  theINCLXXModel->SetMaxEnergy(3.0*CLHEP::GeV);
+	  theInelasticProcess->RegisterMe(theINCLXXModel);
+	  FTFP_model->SetMinEnergy(3.0*CLHEP::GeV);
+	  theInelasticProcess->RegisterMe(FTFP_model);
+	  
+ 	  pmanager->AddDiscreteProcess(theInelasticProcess);
+ 	}
+       
+      else if (particleName == "triton")
+ 	{
+ 	  pmanager->AddDiscreteProcess(theElasticProcess);
+	  G4TritonInelasticProcess* theInelasticProcess =
+ 	    new G4TritonInelasticProcess();
+
+	  // Use Binary Light Ion Reaction for low energy
+	  // and FTFP for high energy - SS
+	  G4BinaryLightIonReaction* ionModel = new G4BinaryLightIonReaction();
+	  ionModel->SetMaxEnergy(10.0*CLHEP::GeV);
+	  theInelasticProcess->RegisterMe(ionModel);
+	  FTFP_model->SetMinEnergy(3.0*CLHEP::GeV);
+	  theInelasticProcess->RegisterMe(FTFP_model);
+
+ 	  pmanager->AddDiscreteProcess(theInelasticProcess);
+ 	}
       
-      else if (particleName == "triton") 
-	{
-	  pmanager->AddDiscreteProcess(theElasticProcess);
-	  G4TritonInelasticProcess* theInelasticProcess = 
-	    new G4TritonInelasticProcess();
-	  G4LETritonInelastic* theLEInelasticModel = 
-	    new G4LETritonInelastic;
-	  theInelasticProcess->RegisterMe(theLEInelasticModel);
-	  pmanager->AddDiscreteProcess(theInelasticProcess);
-	}
-
-      else if (particleName == "alpha") 
-	{
-	  pmanager->AddDiscreteProcess(theElasticProcess);
-	  G4AlphaInelasticProcess* theInelasticProcess = 
-	    new G4AlphaInelasticProcess();
-	  G4LEAlphaInelastic* theLEInelasticModel = 
-	    new G4LEAlphaInelastic;
-	  theInelasticProcess->RegisterMe(theLEInelasticModel);
-	  pmanager->AddDiscreteProcess(theInelasticProcess);
-	}
-
+      else if (particleName == "alpha")
+ 	{
+ 	  pmanager->AddDiscreteProcess(theElasticProcess);
+	  G4AlphaInelasticProcess* theInelasticProcess =
+ 	    new G4AlphaInelasticProcess();
+
+	  // Use Binary for low energy and FTFP for high energy - SS
+	  G4BinaryCascade* theBinaryModel = new G4BinaryCascade();
+	  theBinaryModel->SetMaxEnergy(4.0*CLHEP::GeV);
+	  theInelasticProcess->RegisterMe(theBinaryModel);
+	  FTFP_model->SetMinEnergy(2.0*CLHEP::GeV);
+	  theInelasticProcess->RegisterMe(FTFP_model);
+
+ 	  pmanager->AddDiscreteProcess(theInelasticProcess);
+ 	}
     }
 }
 
-//=================================
-// Added by JLR 2005-07-05
-//=================================
-// Sets secondary hadronic interaction model 
-// Note: this is currently only implemented for
-// protons and neutrons -- not pions.
-// Gheisha = Original Geant4 default 
-// Bertini = Bertini intra-nuclear cascade model
-// Binary  = Binary intra-nuclear cascade model
 void WCSimPhysicsList::SetSecondaryHad(G4String hadval)
 {
+
   SecondaryHadModel = hadval;
 
```

Comment:
From http://geant4.cern.ch/support/ReleaseNotes4.9.6.html 
"Physics lists which depend on the CHIPS package or on the parameterized (Gheisha-like) models (LEP/HEP) are deprecated and will be removed in Geant4 10."